apps/system/js/bluetooth.js

```diff
@@ -221,15 +228,6 @@ var Bluetooth = {
   },
 
   /**
-   * called by external for re-use adapter.
-   *
-   * @public
-   */
-  getAdapter: function bt_getAdapter() {
```

Comment:
It was only used in BluetoothTransfer. Since we have changed to event based, we can remove it now